ecosystem/ffmpeg_plugin/mtl_st20p_rx.c

```diff
@@ -148,6 +164,21 @@ static int mtl_st20p_read_header(AVFormatContext* ctx) {
   dbg(ctx, "%s, fb_cnt: %d\n", __func__, s->fb_cnt);
   ops_rx.framebuff_cnt = s->fb_cnt;
 
+#ifdef MTL_GPU_DIRECT_ENABLED
+  if (s->gpu_direct_enabled) {
+    // create contex for one gpu device
+    GpuContext gpu_ctx = {0};
```

Comment:
```suggestion
    GpuContext gpu_ctx = { 0 };
```

Reply:
This will trigger lint as non compliant tried it already

Reply:
Bad, bad linter.